pkg/scheduler/backend/queue/scheduling_queue.go

```diff
@@ -771,6 +774,16 @@ func (p *PriorityQueue) AddUnschedulableIfNotPresent(logger klog.Logger, pInfo *
 		return fmt.Errorf("Pod %v is already present in the backoff queue", klog.KObj(pod))
 	}
 
+	if len(pInfo.UnschedulablePlugins) == 0 && len(pInfo.PendingPlugins) == 0 {
+		// This Pod came back because of some unexpected errors (e.g., a network issue).
+		pInfo.ErrorCount++
```

Comment:
Since it's not cumulative count, maybe it should be called `ConsecutiveErrors` or `ConsecutiveErrorsCount`?